crates/forge/tests/integration/deploy.rs

```diff
@@ -82,3 +82,33 @@ fn deploy_syscall_check() {
 
     assert_passed(&result);
 }
+
+#[test]
+fn constructor_retdata() {
+    let test = test_case!(
+        indoc!(
+            r#"
+        use result::ResultTrait;
+        use snforge_std::{ declare, ContractClass, ContractClassTrait };
+        use array::ArrayTrait;
+
+        #[test]
+        fn constructor_retdata() {
```

Comment:
Can we add one more test for returning a structure? Or one felt?

Reply:
[Add retdata tests](https://github.com/foundry-rs/starknet-foundry/pull/1957/commits/7ac9dc842bebed3b1c25380f9411856dc8fcc0a1)